AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.ts

```diff
@@ -161,10 +163,22 @@ export class AddParticipantComponent extends BookingBaseComponent implements OnI
         if (this.editMode) {
             if (this.searchEmail && this.participantDetails) {
                 this.setParticipantEmail();
+                this.subcribeForSeachEmailEvents();
             }
         }
     }
 
+    subcribeForSeachEmailEvents() {
+        this.searchEmail.notFoundEmailEvent$.subscribe(notFound => {
```

Comment:
I think the email component should maintain the error state itself - i.e., errorAlternativeEmail & errorJohAccountNotFound should exist in there and not in this component. This component should perhaps just assign a local variable to something like `this.searchEmail.errorAlternativeEmail$' and expose this to the UI.